cptv-reprocess.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/python
+
+import argparse
+
+from api import API
+
+
+class CPTVReprocessor:
```

Comment:
Why is this class necessary? This could just be a single `reprocess` function which takes the arguments which are currently being passed to `__init__`.